Makefile

```diff
@@ -6,7 +6,7 @@ help:
 	@echo "clean-pyc    Remove Python file artifacts"
 	@echo "clean-test   Remove test and coverage artifacts"
 	@echo "lint         Check style with Flake8 and Pylint"
-	@echo "test         Run tests quickly with Python 3.5"
+	@echo "test         Run tests quickly with Python 3.11"
 	@echo "test-all     Run tests on every Python version with tox"
 	@echo "coverage     Check code coverage quickly with Python 3"
 	@echo "dist         Package"
```

Comment:
Not all make targets are listed here in the help e.g. all, test-quick ... are missing.

Reply:
done